source/sass/components/_footer-custom-select.scss

```diff
@@ -0,0 +1,36 @@
+.footer-links {
+  select {
+    &.form-control {
+      $background-size: 24px;
+      $padding-x: 12px;
+      $padding-right: $padding-x * 2 + $background-size;
+
+      -moz-appearance: none;
+      -webkit-appearance: none;
+      background-image: url("data:image/svg+xml,%3Csvg width='24' height='24' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='M4 8l8.05 8.05L20.1 8' stroke='%23FFF' stroke-width='2' fill='none' fill-rule='evenodd' stroke-linecap='round' stroke-linejoin='round'/%3E%3C/svg%3E");
```

Comment:
This looks like it's the same SVG as the one added to the svg fragment file... if so, this is going to desync at some point: is there a way for us to make sure both files point to the same thing instead?

Reply:
These are actually unrelated, one is a globe, this one is the down chevron (coming from https://github.com/mozilla/foundation.mozilla.org/blob/master/source/images/glyphs/down-chevron-dark-theme.svg)